README.md

```diff
@@ -5,23 +5,291 @@
-use of the [istio/installer](https://github.com/istio/installer) charts.
-1.   Provide validation of all parameters.
+## Introduction
+
+The Istio operator CLI is now stable enough for developers to evaluate and experiment with. We welcome your
+contributions - pick an [unassigned open issue](https://github.com/istio/istio/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fenvironments%2Foperator+no%3Aassignee),
+create a feature proposal, or come to the weekly Environments Working Group meeting if you have ideas.
+
+-  [Bugs and feature requests](https://github.com/istio/operator/blob/master/BUGS-AND-FEATURE-REQUESTS.md)
+-  [Contributing guidelines](https://github.com/istio/operator/blob/master/CONTRIBUTING.md)
+-  [Working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md)
+
+### Background
+
+We reorganized the current [helm installation parameters](https://istio.io/docs/reference/config/installation-options/) into two groups:
+
+-  The new [platform level installation API](https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto), for managing
+k8s settings like resources, auto scaling, pod disruption budgets etc. 
+-  The configuration API that currently uses the
+[helm installation parameters](https://istio.io/docs/reference/config/installation-options/) for backwards
+compatibility. This API is for managing This API the Istio control plane configuration settings.
+
+Some parameters will temporarily exist in both APIs - for example, setting k8s resources currently can be done through
+either API above. However, the Istio community recommends using the first API as it is more consistent, is validated,
+and will naturally follow the graduation process for APIs while the same parameters in the configuration API are planned
+for deprecation.
+
+We currently provide pre-configured helm values sets for different scenarios as configuration 
+[profiles](https://istio.io/docs/setup/kubernetes/additional-setup/config-profiles/), which act as a starting point for
+an Istio install and can be customized by creating customization overlay files or passing parameters when
+calling helm. Similarly, the operator API uses the same profiles (expressed internally through the new API), which can be selected
+as a starting point for the installation. For comparison, the following example shows the command needed to install
+Istio using the SDS configuration profile using Helm:
+
+```bash
+helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml | kubectl apply -f -
+```
+
+In the new API, the same profile would be selected through a CustomResource (CR):
+
+```yaml
+# sds-install-cr.yaml
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioControlPlane
+metadata:
+  name: istio-operator-config
+  namespace: istio-system
```

Comment:
i'm confused by this aspect of the API and this is probably a discussion for another thread, however, I'm completely unclear how this permits installation of different components of Istio in different namespaces?

Is the idea the control plane ends up in istio-system?  Some parts of the control plane we want to shave off (citadel for one, and possibly galley) into IIRC istio-base or istio-control.

Reply:
There's an example further down how to specialize namespaces by feature and/or component. But we can take another pass at this doc at a high level to address such gaps.